app/controllers/messages_controller.rb

```diff
@@ -2,84 +2,94 @@ class MessagesController < ApplicationController
 
   before_filter :require_user
   before_filter :require_owner, only: [ :show, :destroy ]
-  
+
   def new
-	  #create the message
-	  @message = Message.new
-	  @title = "New message"
-      @users = User.all
-	  # ideally, we would kick out the current_user however, this generates crashes when there are only two users (function "map" doesn't work on a single object)
-      if params[:message] 
-         @answering = Message.find_by_id(params[:message])
-         @allowed = true
-         if @answering.from_id != current_user.id and @answering.to_id != current_user.id 
-             @allowed = false
-         end
+    #create the message
+    @message = Message.new
+    @title = "New message"
+    @users = User.all
+    # ideally, we would kick out the current_user however, this generates crashes when there are only two users (function "map" doesn't work on a single object)
+    if params[:message] 
+      @answering = Message.find_by_id(params[:message])
+      @allowed = true
+      if @answering.from_id != current_user.id and @answering.to_id != current_user.id 
```

Comment:
Line is too long. [87/80]<br>Use `&&` instead of `and`.<br>Trailing whitespace detected.